2018-06-29 02:28:21.540 31040 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-29 02:28:21.544 31040 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-29 02:29:21.540 31040 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-29 02:29:21.550 31040 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.010s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-29 02:30:02.223 31040 DEBUG oslo_concurrency.lockutils [req-a1e38a84-a7ff-430a-838c-ed2c7e1fff3b f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] Acquired semaphore "dhcp-agent-network-lock-4e75ba7d-7b8a-4fed-a0f2-840daba4d743" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-29 02:30:03.304 31040 DEBUG neutron.agent.dhcp.agent [req-00a4c258-336b-4bae-8e4f-7119c323ac99 - - - - -] Calling driver for network: 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 action: enable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-29 02:30:03.307 31040 DEBUG neutron.agent.linux.utils [req-00a4c258-336b-4bae-8e4f-7119c323ac99 - - - - -] Unable to access /var/lib/neutron/dhcp/4e75ba7d-7b8a-4fed-a0f2-840daba4d743/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-29 02:30:03.309 31040 DEBUG neutron.agent.linux.dhcp [req-00a4c258-336b-4bae-8e4f-7119c323ac99 - - - - -] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-4e75ba7d-7b8a-4fed-a0f2-840daba4d743 on network 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 does not yet exist. Checking for a reserved port. _setup_reserved_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1267
2018-06-29 02:30:03.310 31040 DEBUG neutron.agent.linux.dhcp [req-00a4c258-336b-4bae-8e4f-7119c323ac99 - - - - -] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-4e75ba7d-7b8a-4fed-a0f2-840daba4d743 on network 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 does not yet exist. Creating new one. _setup_new_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1288
2018-06-29 02:30:06.347 31040 DEBUG neutron.agent.linux.utils [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:30:06.372 31040 DEBUG neutron.agent.linux.utils [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'add', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:30:06.388 31040 DEBUG neutron.agent.linux.utils [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743', 'sysctl', '-w', 'net.ipv4.conf.all.promote_secondaries=1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:30:06.408 31040 DEBUG neutron.agent.linux.utils [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743', 'ip', 'link', 'set', 'lo', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:30:06.422 31040 DEBUG neutron.agent.linux.utils [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743', 'sysctl', '-w', 'net.ipv6.conf.default.accept_ra=0'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:30:06.438 31040 DEBUG neutron.agent.linux.utils [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743', 'ip', 'link', 'set', 'tap2bfa1d2e-58', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:30:06.467 31040 DEBUG neutron.agent.linux.utils [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743', 'ip', '-o', 'link', 'show', 'tap2bfa1d2e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:30:06.484 31040 DEBUG neutron.agent.linux.utils [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-06-29 02:30:06.689 31040 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=None, port=tap2bfa1d2e-58) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-29 02:30:06.690 31040 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Transaction caused no change do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:114
2018-06-29 02:30:06.691 31040 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): AddPortCommand(bridge=br-int, may_exist=False, port=tap2bfa1d2e-58) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-29 02:30:06.693 31040 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=1): DbSetCommand(table=Interface, col_values=(('type', 'internal'), ('external_ids', {'iface-id': u'2bfa1d2e-5848-4660-8d69-d8ecd4d11bc2', 'iface-status': 'active', 'attached-mac': u'fa:16:3e:27:ee:32'})), record=tap2bfa1d2e-58) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-29 02:30:06.734 31040 DEBUG neutron.agent.linux.utils [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tap2bfa1d2e-58', 'address', 'fa:16:3e:27:ee:32'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:30:06.749 31040 DEBUG neutron.agent.linux.utils [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:30:06.761 31040 DEBUG neutron.agent.linux.utils [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tap2bfa1d2e-58', 'netns', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:30:06.781 31040 DEBUG neutron.agent.linux.utils [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743', 'ip', 'link', 'set', 'tap2bfa1d2e-58', 'mtu', '1450'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:30:06.797 31040 DEBUG neutron.agent.linux.utils [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743', 'ip', 'link', 'set', 'tap2bfa1d2e-58', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:30:06.809 31040 DEBUG oslo_concurrency.lockutils [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Acquired semaphore "iptables-qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-29 02:30:06.811 31040 DEBUG neutron.agent.linux.utils [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743', 'iptables-save'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:30:06.837 31040 DEBUG neutron.agent.linux.utils [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743', 'iptables-restore', '-n'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:30:06.853 31040 DEBUG neutron.agent.linux.iptables_manager [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] IPTablesManager.apply completed with success. 56 iptables commands were issued _apply_synchronized /usr/lib/python2.7/site-packages/neutron/agent/linux/iptables_manager.py:601
2018-06-29 02:30:06.854 31040 DEBUG oslo_concurrency.lockutils [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Releasing semaphore "iptables-qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-29 02:30:06.855 31040 DEBUG neutron.agent.linux.utils [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743', 'ip', '-o', 'link', 'show', 'tap2bfa1d2e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:30:06.869 31040 DEBUG neutron.agent.linux.utils [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743', 'ip', 'addr', 'show', 'tap2bfa1d2e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:30:06.890 31040 DEBUG neutron.agent.linux.utils [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743', 'ip', '-4', 'addr', 'add', '11.156.1.2/24', 'scope', 'global', 'dev', 'tap2bfa1d2e-58', 'brd', '11.156.1.255'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:30:06.904 31040 DEBUG neutron.agent.linux.utils [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743', 'ip', '-4', 'route', 'list', 'dev', 'tap2bfa1d2e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:30:06.918 31040 DEBUG neutron.agent.linux.dhcp [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Setting IPv4 gateway for dhcp netns on net 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 to 11.156.1.1 _set_default_route_ip_version /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1164
2018-06-29 02:30:06.920 31040 DEBUG neutron.agent.linux.utils [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743', 'ip', '-4', 'route', 'replace', 'default', 'via', '11.156.1.1', 'dev', 'tap2bfa1d2e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:30:06.934 31040 DEBUG neutron.agent.linux.utils [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743', 'ip', '-6', 'route', 'list', 'dev', 'tap2bfa1d2e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:30:06.946 31040 DEBUG neutron.agent.linux.dhcp [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Cleaning stale devices for network 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 _cleanup_stale_devices /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1369
2018-06-29 02:30:06.947 31040 DEBUG neutron.agent.linux.utils [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:30:06.958 31040 DEBUG neutron.agent.linux.utils [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743', 'find', '/sys/class/net', '-maxdepth', '1', '-type', 'l', '-printf', '%f '] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:30:06.977 31040 DEBUG neutron.agent.linux.dhcp [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Building initial lease file: /var/lib/neutron/dhcp/4e75ba7d-7b8a-4fed-a0f2-840daba4d743/leases _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:623
2018-06-29 02:30:06.979 31040 DEBUG neutron.agent.linux.dhcp [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Done building initial lease file /var/lib/neutron/dhcp/4e75ba7d-7b8a-4fed-a0f2-840daba4d743/leases with contents:
1530318606 fa:16:3e:27:ee:32 11.156.1.2 * *
 _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:651
2018-06-29 02:30:06.980 31040 DEBUG neutron.agent.linux.dhcp [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Building host file: /var/lib/neutron/dhcp/4e75ba7d-7b8a-4fed-a0f2-840daba4d743/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-29 02:30:06.982 31040 DEBUG neutron.agent.linux.dhcp [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Done building host file /var/lib/neutron/dhcp/4e75ba7d-7b8a-4fed-a0f2-840daba4d743/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-29 02:30:06.985 31040 DEBUG neutron.agent.linux.utils [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Unable to access /var/lib/neutron/dhcp/4e75ba7d-7b8a-4fed-a0f2-840daba4d743/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-29 02:30:06.987 31040 DEBUG neutron.agent.linux.utils [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743', 'dnsmasq', '--no-hosts', '--no-resolv', '--strict-order', '--except-interface=lo', '--pid-file=/var/lib/neutron/dhcp/4e75ba7d-7b8a-4fed-a0f2-840daba4d743/pid', '--dhcp-hostsfile=/var/lib/neutron/dhcp/4e75ba7d-7b8a-4fed-a0f2-840daba4d743/host', '--addn-hosts=/var/lib/neutron/dhcp/4e75ba7d-7b8a-4fed-a0f2-840daba4d743/addn_hosts', '--dhcp-optsfile=/var/lib/neutron/dhcp/4e75ba7d-7b8a-4fed-a0f2-840daba4d743/opts', '--dhcp-leasefile=/var/lib/neutron/dhcp/4e75ba7d-7b8a-4fed-a0f2-840daba4d743/leases', '--dhcp-match=set:ipxe,175', '--bind-interfaces', '--interface=tap2bfa1d2e-58', '--dhcp-range=set:tag0,11.156.1.0,static,255.255.255.0,86400s', '--dhcp-option-force=option:mtu,1450', '--dhcp-lease-max=256', '--conf-file=', '--domain=openstacklocal'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:30:07.020 31040 DEBUG neutron.agent.linux.utils [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Unable to access /var/lib/neutron/external/pids/4e75ba7d-7b8a-4fed-a0f2-840daba4d743.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-29 02:30:07.023 31040 DEBUG neutron.agent.linux.utils [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Unable to access /var/lib/neutron/external/pids/4e75ba7d-7b8a-4fed-a0f2-840daba4d743.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-29 02:30:07.025 31040 DEBUG neutron.agent.linux.external_process [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] No process started for 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-29 02:30:07.026 31040 DEBUG oslo_concurrency.lockutils [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Releasing semaphore "dhcp-agent-network-lock-4e75ba7d-7b8a-4fed-a0f2-840daba4d743" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-29 02:30:07.027 31040 DEBUG oslo_concurrency.lockutils [req-a1e38a84-a7ff-430a-838c-ed2c7e1fff3b f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] Acquired semaphore "dhcp-agent-network-lock-4e75ba7d-7b8a-4fed-a0f2-840daba4d743" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-29 02:30:07.606 31040 DEBUG neutron.agent.dhcp.agent [req-b5f273d2-e27d-4375-a5b4-6221ee6d29fb - - - - -] Calling driver for network: 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-29 02:30:07.608 31040 DEBUG neutron.agent.linux.dhcp [req-b5f273d2-e27d-4375-a5b4-6221ee6d29fb - - - - -] Building host file: /var/lib/neutron/dhcp/4e75ba7d-7b8a-4fed-a0f2-840daba4d743/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-29 02:30:07.610 31040 DEBUG neutron.agent.linux.dhcp [req-b5f273d2-e27d-4375-a5b4-6221ee6d29fb - - - - -] Done building host file /var/lib/neutron/dhcp/4e75ba7d-7b8a-4fed-a0f2-840daba4d743/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-29 02:30:07.623 31040 DEBUG neutron.agent.linux.utils [req-b5f273d2-e27d-4375-a5b4-6221ee6d29fb - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '2759'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:30:07.641 31040 DEBUG neutron.agent.linux.dhcp [req-b5f273d2-e27d-4375-a5b4-6221ee6d29fb - - - - -] Reloading allocations for network: 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-29 02:30:07.644 31040 DEBUG neutron.agent.linux.utils [req-b5f273d2-e27d-4375-a5b4-6221ee6d29fb - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743', 'ip', '-4', 'route', 'list', 'dev', 'tap2bfa1d2e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:30:07.661 31040 DEBUG neutron.agent.linux.utils [req-b5f273d2-e27d-4375-a5b4-6221ee6d29fb - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743', 'ip', '-6', 'route', 'list', 'dev', 'tap2bfa1d2e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:30:07.675 31040 DEBUG neutron.agent.linux.utils [req-b5f273d2-e27d-4375-a5b4-6221ee6d29fb - - - - -] Unable to access /var/lib/neutron/external/pids/4e75ba7d-7b8a-4fed-a0f2-840daba4d743.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-29 02:30:07.676 31040 DEBUG neutron.agent.linux.utils [req-b5f273d2-e27d-4375-a5b4-6221ee6d29fb - - - - -] Unable to access /var/lib/neutron/external/pids/4e75ba7d-7b8a-4fed-a0f2-840daba4d743.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-29 02:30:07.676 31040 DEBUG neutron.agent.linux.external_process [req-b5f273d2-e27d-4375-a5b4-6221ee6d29fb - - - - -] No process started for 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-29 02:30:07.677 31040 DEBUG oslo_concurrency.lockutils [req-b5f273d2-e27d-4375-a5b4-6221ee6d29fb - - - - -] Releasing semaphore "dhcp-agent-network-lock-4e75ba7d-7b8a-4fed-a0f2-840daba4d743" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-29 02:30:07.679 31040 DEBUG oslo_concurrency.lockutils [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Acquired semaphore "dhcp-agent-network-lock-4e75ba7d-7b8a-4fed-a0f2-840daba4d743" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-29 02:30:07.680 31040 DEBUG neutron.agent.dhcp.agent [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Discarding stale port update: admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-29T00:30:03Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-4e75ba7d-7b8a-4fed-a0f2-840daba4d743, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'f5f0c563-7abb-4002-a413-730683323181', u'ip_address': u'11.156.1.2'}], id=2bfa1d2e-5848-4660-8d69-d8ecd4d11bc2, mac_address=fa:16:3e:27:ee:32, name=, network_id=4e75ba7d-7b8a-4fed-a0f2-840daba4d743, port_security_enabled=False, project_id=2bae9361f8ba48608796babaf1c6b310, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=2bae9361f8ba48608796babaf1c6b310, updated_at=2018-06-29T00:30:04Z port_update_end /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:409
2018-06-29 02:30:07.681 31040 DEBUG oslo_concurrency.lockutils [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Releasing semaphore "dhcp-agent-network-lock-4e75ba7d-7b8a-4fed-a0f2-840daba4d743" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-29 02:30:07.683 31040 DEBUG oslo_concurrency.lockutils [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Acquired semaphore "dhcp-agent-network-lock-4e75ba7d-7b8a-4fed-a0f2-840daba4d743" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-29 02:30:07.683 31040 DEBUG neutron.agent.dhcp.agent [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Discarding stale port update: admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-06-29T00:30:03Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-4e75ba7d-7b8a-4fed-a0f2-840daba4d743, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'f5f0c563-7abb-4002-a413-730683323181', u'ip_address': u'11.156.1.2'}], id=2bfa1d2e-5848-4660-8d69-d8ecd4d11bc2, mac_address=fa:16:3e:27:ee:32, name=, network_id=4e75ba7d-7b8a-4fed-a0f2-840daba4d743, port_security_enabled=False, project_id=2bae9361f8ba48608796babaf1c6b310, revision_number=4, security_groups=[], status=DOWN, tags=[], tenant_id=2bae9361f8ba48608796babaf1c6b310, updated_at=2018-06-29T00:30:05Z port_update_end /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:409
2018-06-29 02:30:07.684 31040 DEBUG oslo_concurrency.lockutils [req-b0d41fe8-c01a-4a34-b1aa-d8eae6df0983 - - - - -] Releasing semaphore "dhcp-agent-network-lock-4e75ba7d-7b8a-4fed-a0f2-840daba4d743" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-29 02:30:21.540 31040 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-29 02:30:21.543 31040 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-29 02:30:44.301 31040 DEBUG oslo_concurrency.lockutils [req-c60f4ff6-117b-4642-a932-bb5de826c4db f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] Acquired semaphore "dhcp-agent-network-lock-4e75ba7d-7b8a-4fed-a0f2-840daba4d743" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-29 02:30:44.302 31040 INFO neutron.agent.dhcp.agent [req-c60f4ff6-117b-4642-a932-bb5de826c4db f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-29T00:30:42Z, description=, device_id=446272e2-d85c-4c39-b8de-6aec7a39c9fe, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'f5f0c563-7abb-4002-a413-730683323181', u'ip_address': u'11.156.1.1'}], id=28d5df8e-ea9d-42fb-b18e-4df653fc3d11, mac_address=fa:16:3e:17:b9:55, name=, network_id=4e75ba7d-7b8a-4fed-a0f2-840daba4d743, port_security_enabled=False, project_id=2bae9361f8ba48608796babaf1c6b310, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=2bae9361f8ba48608796babaf1c6b310, updated_at=2018-06-29T00:30:43Z
2018-06-29 02:30:44.303 31040 DEBUG neutron.agent.dhcp.agent [req-c60f4ff6-117b-4642-a932-bb5de826c4db f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] Calling driver for network: 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-29 02:30:44.304 31040 DEBUG neutron.agent.linux.dhcp [req-c60f4ff6-117b-4642-a932-bb5de826c4db f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] Building host file: /var/lib/neutron/dhcp/4e75ba7d-7b8a-4fed-a0f2-840daba4d743/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-29 02:30:44.308 31040 DEBUG neutron.agent.linux.dhcp [req-c60f4ff6-117b-4642-a932-bb5de826c4db f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] Done building host file /var/lib/neutron/dhcp/4e75ba7d-7b8a-4fed-a0f2-840daba4d743/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-29 02:30:44.314 31040 DEBUG neutron.agent.linux.utils [req-c60f4ff6-117b-4642-a932-bb5de826c4db f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '2759'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:30:44.335 31040 DEBUG neutron.agent.linux.dhcp [req-c60f4ff6-117b-4642-a932-bb5de826c4db f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] Reloading allocations for network: 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-29 02:30:44.336 31040 DEBUG neutron.agent.linux.utils [req-c60f4ff6-117b-4642-a932-bb5de826c4db f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743', 'ip', '-4', 'route', 'list', 'dev', 'tap2bfa1d2e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:30:44.354 31040 DEBUG neutron.agent.linux.utils [req-c60f4ff6-117b-4642-a932-bb5de826c4db f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743', 'ip', '-6', 'route', 'list', 'dev', 'tap2bfa1d2e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:30:44.370 31040 DEBUG neutron.agent.linux.utils [req-c60f4ff6-117b-4642-a932-bb5de826c4db f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] Unable to access /var/lib/neutron/external/pids/4e75ba7d-7b8a-4fed-a0f2-840daba4d743.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-29 02:30:44.371 31040 DEBUG neutron.agent.linux.utils [req-c60f4ff6-117b-4642-a932-bb5de826c4db f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] Unable to access /var/lib/neutron/external/pids/4e75ba7d-7b8a-4fed-a0f2-840daba4d743.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-29 02:30:44.372 31040 DEBUG neutron.agent.linux.external_process [req-c60f4ff6-117b-4642-a932-bb5de826c4db f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] No process started for 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-29 02:30:44.373 31040 DEBUG oslo_concurrency.lockutils [req-c60f4ff6-117b-4642-a932-bb5de826c4db f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] Releasing semaphore "dhcp-agent-network-lock-4e75ba7d-7b8a-4fed-a0f2-840daba4d743" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-29 02:30:45.435 31040 DEBUG oslo_concurrency.lockutils [req-c60f4ff6-117b-4642-a932-bb5de826c4db f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] Acquired semaphore "dhcp-agent-network-lock-4e75ba7d-7b8a-4fed-a0f2-840daba4d743" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-29 02:30:45.437 31040 INFO neutron.agent.dhcp.agent [req-c60f4ff6-117b-4642-a932-bb5de826c4db f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-29T00:30:42Z, description=, device_id=446272e2-d85c-4c39-b8de-6aec7a39c9fe, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'f5f0c563-7abb-4002-a413-730683323181', u'ip_address': u'11.156.1.1'}], id=28d5df8e-ea9d-42fb-b18e-4df653fc3d11, mac_address=fa:16:3e:17:b9:55, name=, network_id=4e75ba7d-7b8a-4fed-a0f2-840daba4d743, port_security_enabled=False, project_id=2bae9361f8ba48608796babaf1c6b310, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=2bae9361f8ba48608796babaf1c6b310, updated_at=2018-06-29T00:30:43Z
2018-06-29 02:30:45.439 31040 DEBUG neutron.agent.dhcp.agent [req-c60f4ff6-117b-4642-a932-bb5de826c4db f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] Calling driver for network: 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-29 02:30:45.441 31040 DEBUG neutron.agent.linux.dhcp [req-c60f4ff6-117b-4642-a932-bb5de826c4db f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] Building host file: /var/lib/neutron/dhcp/4e75ba7d-7b8a-4fed-a0f2-840daba4d743/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-29 02:30:45.444 31040 DEBUG neutron.agent.linux.dhcp [req-c60f4ff6-117b-4642-a932-bb5de826c4db f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] Done building host file /var/lib/neutron/dhcp/4e75ba7d-7b8a-4fed-a0f2-840daba4d743/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-29 02:30:45.449 31040 DEBUG neutron.agent.linux.utils [req-c60f4ff6-117b-4642-a932-bb5de826c4db f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '2759'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:30:45.465 31040 DEBUG neutron.agent.linux.dhcp [req-c60f4ff6-117b-4642-a932-bb5de826c4db f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] Reloading allocations for network: 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-29 02:30:45.467 31040 DEBUG neutron.agent.linux.utils [req-c60f4ff6-117b-4642-a932-bb5de826c4db f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743', 'ip', '-4', 'route', 'list', 'dev', 'tap2bfa1d2e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:30:45.482 31040 DEBUG neutron.agent.linux.utils [req-c60f4ff6-117b-4642-a932-bb5de826c4db f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743', 'ip', '-6', 'route', 'list', 'dev', 'tap2bfa1d2e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:30:45.499 31040 DEBUG neutron.agent.linux.utils [req-c60f4ff6-117b-4642-a932-bb5de826c4db f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] Unable to access /var/lib/neutron/external/pids/4e75ba7d-7b8a-4fed-a0f2-840daba4d743.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-29 02:30:45.500 31040 DEBUG neutron.agent.linux.utils [req-c60f4ff6-117b-4642-a932-bb5de826c4db f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] Unable to access /var/lib/neutron/external/pids/4e75ba7d-7b8a-4fed-a0f2-840daba4d743.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-29 02:30:45.500 31040 DEBUG neutron.agent.linux.external_process [req-c60f4ff6-117b-4642-a932-bb5de826c4db f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] No process started for 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-29 02:30:45.501 31040 DEBUG oslo_concurrency.lockutils [req-c60f4ff6-117b-4642-a932-bb5de826c4db f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] Releasing semaphore "dhcp-agent-network-lock-4e75ba7d-7b8a-4fed-a0f2-840daba4d743" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-29 02:30:48.370 31040 DEBUG oslo_concurrency.lockutils [req-6e77c1ac-7338-4632-a69d-326abdf81492 - - - - -] Acquired semaphore "dhcp-agent-network-lock-4e75ba7d-7b8a-4fed-a0f2-840daba4d743" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-29 02:30:48.371 31040 INFO neutron.agent.dhcp.agent [req-6e77c1ac-7338-4632-a69d-326abdf81492 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-29T00:30:42Z, description=, device_id=446272e2-d85c-4c39-b8de-6aec7a39c9fe, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'f5f0c563-7abb-4002-a413-730683323181', u'ip_address': u'11.156.1.1'}], id=28d5df8e-ea9d-42fb-b18e-4df653fc3d11, mac_address=fa:16:3e:17:b9:55, name=, network_id=4e75ba7d-7b8a-4fed-a0f2-840daba4d743, port_security_enabled=False, project_id=2bae9361f8ba48608796babaf1c6b310, revision_number=4, security_groups=[], status=DOWN, tags=[], tenant_id=2bae9361f8ba48608796babaf1c6b310, updated_at=2018-06-29T00:30:44Z
2018-06-29 02:30:48.372 31040 DEBUG neutron.agent.dhcp.agent [req-6e77c1ac-7338-4632-a69d-326abdf81492 - - - - -] Calling driver for network: 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-29 02:30:48.374 31040 DEBUG neutron.agent.linux.dhcp [req-6e77c1ac-7338-4632-a69d-326abdf81492 - - - - -] Building host file: /var/lib/neutron/dhcp/4e75ba7d-7b8a-4fed-a0f2-840daba4d743/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-29 02:30:48.378 31040 DEBUG neutron.agent.linux.dhcp [req-6e77c1ac-7338-4632-a69d-326abdf81492 - - - - -] Done building host file /var/lib/neutron/dhcp/4e75ba7d-7b8a-4fed-a0f2-840daba4d743/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-29 02:30:48.383 31040 DEBUG neutron.agent.linux.utils [req-6e77c1ac-7338-4632-a69d-326abdf81492 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '2759'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:30:48.404 31040 DEBUG neutron.agent.linux.dhcp [req-6e77c1ac-7338-4632-a69d-326abdf81492 - - - - -] Reloading allocations for network: 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-29 02:30:48.406 31040 DEBUG neutron.agent.linux.utils [req-6e77c1ac-7338-4632-a69d-326abdf81492 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743', 'ip', '-4', 'route', 'list', 'dev', 'tap2bfa1d2e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:30:48.423 31040 DEBUG neutron.agent.linux.utils [req-6e77c1ac-7338-4632-a69d-326abdf81492 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743', 'ip', '-6', 'route', 'list', 'dev', 'tap2bfa1d2e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:30:48.441 31040 DEBUG neutron.agent.linux.utils [req-6e77c1ac-7338-4632-a69d-326abdf81492 - - - - -] Unable to access /var/lib/neutron/external/pids/4e75ba7d-7b8a-4fed-a0f2-840daba4d743.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-29 02:30:48.442 31040 DEBUG neutron.agent.linux.utils [req-6e77c1ac-7338-4632-a69d-326abdf81492 - - - - -] Unable to access /var/lib/neutron/external/pids/4e75ba7d-7b8a-4fed-a0f2-840daba4d743.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-29 02:30:48.444 31040 DEBUG neutron.agent.linux.external_process [req-6e77c1ac-7338-4632-a69d-326abdf81492 - - - - -] No process started for 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-29 02:30:48.446 31040 DEBUG oslo_concurrency.lockutils [req-6e77c1ac-7338-4632-a69d-326abdf81492 - - - - -] Releasing semaphore "dhcp-agent-network-lock-4e75ba7d-7b8a-4fed-a0f2-840daba4d743" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-29 02:30:49.065 31040 DEBUG oslo_concurrency.lockutils [req-6e77c1ac-7338-4632-a69d-326abdf81492 - - - - -] Acquired semaphore "dhcp-agent-network-lock-4e75ba7d-7b8a-4fed-a0f2-840daba4d743" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-29 02:30:49.066 31040 INFO neutron.agent.dhcp.agent [req-6e77c1ac-7338-4632-a69d-326abdf81492 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-06-29T00:30:42Z, description=, device_id=446272e2-d85c-4c39-b8de-6aec7a39c9fe, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'f5f0c563-7abb-4002-a413-730683323181', u'ip_address': u'11.156.1.1'}], id=28d5df8e-ea9d-42fb-b18e-4df653fc3d11, mac_address=fa:16:3e:17:b9:55, name=, network_id=4e75ba7d-7b8a-4fed-a0f2-840daba4d743, port_security_enabled=False, project_id=2bae9361f8ba48608796babaf1c6b310, revision_number=6, security_groups=[], status=DOWN, tags=[], tenant_id=2bae9361f8ba48608796babaf1c6b310, updated_at=2018-06-29T00:30:48Z
2018-06-29 02:30:49.068 31040 DEBUG neutron.agent.dhcp.agent [req-6e77c1ac-7338-4632-a69d-326abdf81492 - - - - -] Calling driver for network: 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-29 02:30:49.070 31040 DEBUG neutron.agent.linux.dhcp [req-6e77c1ac-7338-4632-a69d-326abdf81492 - - - - -] Building host file: /var/lib/neutron/dhcp/4e75ba7d-7b8a-4fed-a0f2-840daba4d743/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-29 02:30:49.075 31040 DEBUG neutron.agent.linux.dhcp [req-6e77c1ac-7338-4632-a69d-326abdf81492 - - - - -] Done building host file /var/lib/neutron/dhcp/4e75ba7d-7b8a-4fed-a0f2-840daba4d743/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-29 02:30:49.083 31040 DEBUG neutron.agent.linux.utils [req-6e77c1ac-7338-4632-a69d-326abdf81492 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '2759'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:30:49.101 31040 DEBUG neutron.agent.linux.dhcp [req-6e77c1ac-7338-4632-a69d-326abdf81492 - - - - -] Reloading allocations for network: 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-29 02:30:49.105 31040 DEBUG neutron.agent.linux.utils [req-6e77c1ac-7338-4632-a69d-326abdf81492 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743', 'ip', '-4', 'route', 'list', 'dev', 'tap2bfa1d2e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:30:49.123 31040 DEBUG neutron.agent.linux.utils [req-6e77c1ac-7338-4632-a69d-326abdf81492 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743', 'ip', '-6', 'route', 'list', 'dev', 'tap2bfa1d2e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:30:49.136 31040 DEBUG neutron.agent.linux.utils [req-6e77c1ac-7338-4632-a69d-326abdf81492 - - - - -] Unable to access /var/lib/neutron/external/pids/4e75ba7d-7b8a-4fed-a0f2-840daba4d743.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-29 02:30:49.137 31040 DEBUG neutron.agent.linux.utils [req-6e77c1ac-7338-4632-a69d-326abdf81492 - - - - -] Unable to access /var/lib/neutron/external/pids/4e75ba7d-7b8a-4fed-a0f2-840daba4d743.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-29 02:30:49.138 31040 DEBUG neutron.agent.linux.external_process [req-6e77c1ac-7338-4632-a69d-326abdf81492 - - - - -] No process started for 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-29 02:30:49.139 31040 DEBUG oslo_concurrency.lockutils [req-6e77c1ac-7338-4632-a69d-326abdf81492 - - - - -] Releasing semaphore "dhcp-agent-network-lock-4e75ba7d-7b8a-4fed-a0f2-840daba4d743" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-29 02:30:56.626 31040 DEBUG oslo_concurrency.lockutils [req-56015149-8f6e-4166-88d7-dfed98b2775f f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] Acquired semaphore "dhcp-agent-network-lock-3a89a4af-574f-492a-9eb4-eae63d51f625" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-29 02:30:56.627 31040 DEBUG oslo_concurrency.lockutils [req-56015149-8f6e-4166-88d7-dfed98b2775f f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] Releasing semaphore "dhcp-agent-network-lock-3a89a4af-574f-492a-9eb4-eae63d51f625" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-29 02:31:00.140 31040 DEBUG oslo_concurrency.lockutils [req-b4141f19-3a4f-4815-bf6e-e3e3253aaa55 - - - - -] Acquired semaphore "dhcp-agent-network-lock-3a89a4af-574f-492a-9eb4-eae63d51f625" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-29 02:31:00.141 31040 DEBUG oslo_concurrency.lockutils [req-b4141f19-3a4f-4815-bf6e-e3e3253aaa55 - - - - -] Releasing semaphore "dhcp-agent-network-lock-3a89a4af-574f-492a-9eb4-eae63d51f625" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-29 02:31:00.888 31040 DEBUG oslo_concurrency.lockutils [req-b4141f19-3a4f-4815-bf6e-e3e3253aaa55 - - - - -] Acquired semaphore "dhcp-agent-network-lock-3a89a4af-574f-492a-9eb4-eae63d51f625" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-29 02:31:00.889 31040 DEBUG oslo_concurrency.lockutils [req-b4141f19-3a4f-4815-bf6e-e3e3253aaa55 - - - - -] Releasing semaphore "dhcp-agent-network-lock-3a89a4af-574f-492a-9eb4-eae63d51f625" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-29 02:31:21.541 31040 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-29 02:31:21.543 31040 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-29 02:31:23.357 31040 DEBUG oslo_concurrency.lockutils [req-e6390e9b-1ed0-46d3-b05b-409596997884 f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] Acquired semaphore "dhcp-agent-network-lock-4e75ba7d-7b8a-4fed-a0f2-840daba4d743" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-29 02:31:23.358 31040 INFO neutron.agent.dhcp.agent [req-e6390e9b-1ed0-46d3-b05b-409596997884 f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-29T00:31:21Z, description=, device_id=6a2ad285-a338-4ded-974c-602c4f2446fa, device_owner=, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'f5f0c563-7abb-4002-a413-730683323181', u'ip_address': u'11.156.1.9'}], id=dfe1f18f-b9dd-44cb-b172-3eca9376f3e9, mac_address=fa:16:3e:aa:9a:32, name=, network_id=4e75ba7d-7b8a-4fed-a0f2-840daba4d743, port_security_enabled=True, project_id=2bae9361f8ba48608796babaf1c6b310, revision_number=3, security_groups=[u'e64ab033-c82e-4c4a-bc18-2104dc415e0b'], status=DOWN, tags=[], tenant_id=2bae9361f8ba48608796babaf1c6b310, updated_at=2018-06-29T00:31:22Z
2018-06-29 02:31:23.359 31040 DEBUG neutron.agent.dhcp.agent [req-e6390e9b-1ed0-46d3-b05b-409596997884 f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] Calling driver for network: 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-29 02:31:23.361 31040 DEBUG neutron.agent.linux.dhcp [req-e6390e9b-1ed0-46d3-b05b-409596997884 f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] Building host file: /var/lib/neutron/dhcp/4e75ba7d-7b8a-4fed-a0f2-840daba4d743/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-29 02:31:23.368 31040 DEBUG neutron.agent.linux.dhcp [req-e6390e9b-1ed0-46d3-b05b-409596997884 f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] Done building host file /var/lib/neutron/dhcp/4e75ba7d-7b8a-4fed-a0f2-840daba4d743/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-29 02:31:23.374 31040 DEBUG neutron.agent.linux.utils [req-e6390e9b-1ed0-46d3-b05b-409596997884 f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '2759'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:31:23.400 31040 DEBUG neutron.agent.linux.dhcp [req-e6390e9b-1ed0-46d3-b05b-409596997884 f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] Reloading allocations for network: 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-29 02:31:23.402 31040 DEBUG neutron.agent.linux.utils [req-e6390e9b-1ed0-46d3-b05b-409596997884 f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743', 'ip', '-4', 'route', 'list', 'dev', 'tap2bfa1d2e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:31:23.417 31040 DEBUG neutron.agent.linux.utils [req-e6390e9b-1ed0-46d3-b05b-409596997884 f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743', 'ip', '-6', 'route', 'list', 'dev', 'tap2bfa1d2e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:31:23.437 31040 DEBUG neutron.agent.linux.utils [req-e6390e9b-1ed0-46d3-b05b-409596997884 f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] Unable to access /var/lib/neutron/external/pids/4e75ba7d-7b8a-4fed-a0f2-840daba4d743.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-29 02:31:23.439 31040 DEBUG neutron.agent.linux.utils [req-e6390e9b-1ed0-46d3-b05b-409596997884 f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] Unable to access /var/lib/neutron/external/pids/4e75ba7d-7b8a-4fed-a0f2-840daba4d743.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-29 02:31:23.439 31040 DEBUG neutron.agent.linux.external_process [req-e6390e9b-1ed0-46d3-b05b-409596997884 f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] No process started for 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-29 02:31:23.440 31040 DEBUG oslo_concurrency.lockutils [req-e6390e9b-1ed0-46d3-b05b-409596997884 f3b7ff1aadb54be18e0a36a9d47ad8ea 2bae9361f8ba48608796babaf1c6b310 - - -] Releasing semaphore "dhcp-agent-network-lock-4e75ba7d-7b8a-4fed-a0f2-840daba4d743" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-29 02:31:26.676 31040 DEBUG oslo_concurrency.lockutils [req-00a3da0d-2dd1-4fdb-840f-ac1372fee9cd 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Acquired semaphore "dhcp-agent-network-lock-4e75ba7d-7b8a-4fed-a0f2-840daba4d743" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-29 02:31:26.677 31040 INFO neutron.agent.dhcp.agent [req-00a3da0d-2dd1-4fdb-840f-ac1372fee9cd 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-29T00:31:21Z, description=, device_id=6a2ad285-a338-4ded-974c-602c4f2446fa, device_owner=compute:nova, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'f5f0c563-7abb-4002-a413-730683323181', u'ip_address': u'11.156.1.9'}], id=dfe1f18f-b9dd-44cb-b172-3eca9376f3e9, mac_address=fa:16:3e:aa:9a:32, name=, network_id=4e75ba7d-7b8a-4fed-a0f2-840daba4d743, port_security_enabled=True, project_id=2bae9361f8ba48608796babaf1c6b310, revision_number=4, security_groups=[u'e64ab033-c82e-4c4a-bc18-2104dc415e0b'], status=DOWN, tags=[], tenant_id=2bae9361f8ba48608796babaf1c6b310, updated_at=2018-06-29T00:31:25Z
2018-06-29 02:31:26.679 31040 DEBUG neutron.agent.dhcp.agent [req-00a3da0d-2dd1-4fdb-840f-ac1372fee9cd 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Calling driver for network: 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-29 02:31:26.680 31040 DEBUG neutron.agent.linux.dhcp [req-00a3da0d-2dd1-4fdb-840f-ac1372fee9cd 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Building host file: /var/lib/neutron/dhcp/4e75ba7d-7b8a-4fed-a0f2-840daba4d743/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-29 02:31:26.683 31040 DEBUG neutron.agent.linux.dhcp [req-00a3da0d-2dd1-4fdb-840f-ac1372fee9cd 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Done building host file /var/lib/neutron/dhcp/4e75ba7d-7b8a-4fed-a0f2-840daba4d743/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-29 02:31:26.690 31040 DEBUG neutron.agent.linux.utils [req-00a3da0d-2dd1-4fdb-840f-ac1372fee9cd 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Running command (rootwrap daemon): ['kill', '-HUP', '2759'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:31:26.706 31040 DEBUG neutron.agent.linux.dhcp [req-00a3da0d-2dd1-4fdb-840f-ac1372fee9cd 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Reloading allocations for network: 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-29 02:31:26.709 31040 DEBUG neutron.agent.linux.utils [req-00a3da0d-2dd1-4fdb-840f-ac1372fee9cd 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743', 'ip', '-4', 'route', 'list', 'dev', 'tap2bfa1d2e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:31:26.721 31040 DEBUG neutron.agent.linux.utils [req-00a3da0d-2dd1-4fdb-840f-ac1372fee9cd 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743', 'ip', '-6', 'route', 'list', 'dev', 'tap2bfa1d2e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:31:26.733 31040 DEBUG neutron.agent.linux.utils [req-00a3da0d-2dd1-4fdb-840f-ac1372fee9cd 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Unable to access /var/lib/neutron/external/pids/4e75ba7d-7b8a-4fed-a0f2-840daba4d743.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-29 02:31:26.735 31040 DEBUG neutron.agent.linux.utils [req-00a3da0d-2dd1-4fdb-840f-ac1372fee9cd 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Unable to access /var/lib/neutron/external/pids/4e75ba7d-7b8a-4fed-a0f2-840daba4d743.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-29 02:31:26.736 31040 DEBUG neutron.agent.linux.external_process [req-00a3da0d-2dd1-4fdb-840f-ac1372fee9cd 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] No process started for 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-29 02:31:26.737 31040 DEBUG oslo_concurrency.lockutils [req-00a3da0d-2dd1-4fdb-840f-ac1372fee9cd 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Releasing semaphore "dhcp-agent-network-lock-4e75ba7d-7b8a-4fed-a0f2-840daba4d743" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-29 02:31:27.180 31040 DEBUG oslo_concurrency.lockutils [req-00a3da0d-2dd1-4fdb-840f-ac1372fee9cd 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Acquired semaphore "dhcp-agent-network-lock-4e75ba7d-7b8a-4fed-a0f2-840daba4d743" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-29 02:31:27.181 31040 INFO neutron.agent.dhcp.agent [req-00a3da0d-2dd1-4fdb-840f-ac1372fee9cd 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-06-29T00:31:21Z, description=, device_id=6a2ad285-a338-4ded-974c-602c4f2446fa, device_owner=compute:nova, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'f5f0c563-7abb-4002-a413-730683323181', u'ip_address': u'11.156.1.9'}], id=dfe1f18f-b9dd-44cb-b172-3eca9376f3e9, mac_address=fa:16:3e:aa:9a:32, name=, network_id=4e75ba7d-7b8a-4fed-a0f2-840daba4d743, port_security_enabled=True, project_id=2bae9361f8ba48608796babaf1c6b310, revision_number=5, security_groups=[u'e64ab033-c82e-4c4a-bc18-2104dc415e0b'], status=DOWN, tags=[], tenant_id=2bae9361f8ba48608796babaf1c6b310, updated_at=2018-06-29T00:31:26Z
2018-06-29 02:31:27.183 31040 DEBUG neutron.agent.dhcp.agent [req-00a3da0d-2dd1-4fdb-840f-ac1372fee9cd 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Calling driver for network: 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-29 02:31:27.186 31040 DEBUG neutron.agent.linux.dhcp [req-00a3da0d-2dd1-4fdb-840f-ac1372fee9cd 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Building host file: /var/lib/neutron/dhcp/4e75ba7d-7b8a-4fed-a0f2-840daba4d743/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-29 02:31:27.188 31040 DEBUG neutron.agent.linux.dhcp [req-00a3da0d-2dd1-4fdb-840f-ac1372fee9cd 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Done building host file /var/lib/neutron/dhcp/4e75ba7d-7b8a-4fed-a0f2-840daba4d743/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-29 02:31:27.191 31040 DEBUG neutron.agent.linux.utils [req-00a3da0d-2dd1-4fdb-840f-ac1372fee9cd 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Running command (rootwrap daemon): ['kill', '-HUP', '2759'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:31:27.203 31040 DEBUG neutron.agent.linux.dhcp [req-00a3da0d-2dd1-4fdb-840f-ac1372fee9cd 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Reloading allocations for network: 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-29 02:31:27.205 31040 DEBUG neutron.agent.linux.utils [req-00a3da0d-2dd1-4fdb-840f-ac1372fee9cd 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743', 'ip', '-4', 'route', 'list', 'dev', 'tap2bfa1d2e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:31:27.216 31040 DEBUG neutron.agent.linux.utils [req-00a3da0d-2dd1-4fdb-840f-ac1372fee9cd 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743', 'ip', '-6', 'route', 'list', 'dev', 'tap2bfa1d2e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:31:27.229 31040 DEBUG neutron.agent.linux.utils [req-00a3da0d-2dd1-4fdb-840f-ac1372fee9cd 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Unable to access /var/lib/neutron/external/pids/4e75ba7d-7b8a-4fed-a0f2-840daba4d743.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-29 02:31:27.230 31040 DEBUG neutron.agent.linux.utils [req-00a3da0d-2dd1-4fdb-840f-ac1372fee9cd 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Unable to access /var/lib/neutron/external/pids/4e75ba7d-7b8a-4fed-a0f2-840daba4d743.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-29 02:31:27.231 31040 DEBUG neutron.agent.linux.external_process [req-00a3da0d-2dd1-4fdb-840f-ac1372fee9cd 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] No process started for 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-29 02:31:27.232 31040 DEBUG oslo_concurrency.lockutils [req-00a3da0d-2dd1-4fdb-840f-ac1372fee9cd 75fa4703edb245b7af4eedb4f8abb678 0756a888a15d4216915a13bf41cedcba - - -] Releasing semaphore "dhcp-agent-network-lock-4e75ba7d-7b8a-4fed-a0f2-840daba4d743" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-29 02:32:07.002 31040 DEBUG oslo_concurrency.lockutils [req-cd53c8c4-f922-4409-b413-65ad3899d579 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Acquired semaphore "dhcp-agent-network-lock-3a89a4af-574f-492a-9eb4-eae63d51f625" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-29 02:32:07.003 31040 DEBUG oslo_concurrency.lockutils [req-cd53c8c4-f922-4409-b413-65ad3899d579 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Releasing semaphore "dhcp-agent-network-lock-3a89a4af-574f-492a-9eb4-eae63d51f625" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-29 02:32:08.206 31040 DEBUG oslo_concurrency.lockutils [req-cd53c8c4-f922-4409-b413-65ad3899d579 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Acquired semaphore "dhcp-agent-network-lock-3a89a4af-574f-492a-9eb4-eae63d51f625" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-29 02:32:08.207 31040 DEBUG oslo_concurrency.lockutils [req-cd53c8c4-f922-4409-b413-65ad3899d579 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Releasing semaphore "dhcp-agent-network-lock-3a89a4af-574f-492a-9eb4-eae63d51f625" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-29 02:32:21.541 31040 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-29 02:32:21.546 31040 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.005s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-29 02:33:21.541 31040 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-29 02:33:21.545 31040 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-29 02:34:21.542 31040 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-29 02:34:21.547 31040 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.005s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-29 02:35:21.542 31040 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-29 02:35:21.547 31040 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.005s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-29 02:36:21.543 31040 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-29 02:36:21.548 31040 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.006s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-29 02:36:33.058 31040 DEBUG oslo_concurrency.lockutils [req-7a0b6989-8a94-435e-9762-a778d4869432 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Acquired semaphore "dhcp-agent-network-lock-4e75ba7d-7b8a-4fed-a0f2-840daba4d743" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-29 02:36:33.060 31040 DEBUG neutron.agent.dhcp.agent [req-7a0b6989-8a94-435e-9762-a778d4869432 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Calling driver for network: 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-29 02:36:33.063 31040 DEBUG neutron.agent.linux.utils [req-7a0b6989-8a94-435e-9762-a778d4869432 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743', 'dhcp_release', 'tap2bfa1d2e-58', '11.156.1.9', 'fa:16:3e:aa:9a:32'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:36:33.099 31040 DEBUG neutron.agent.linux.dhcp [req-7a0b6989-8a94-435e-9762-a778d4869432 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Building host file: /var/lib/neutron/dhcp/4e75ba7d-7b8a-4fed-a0f2-840daba4d743/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-29 02:36:33.106 31040 DEBUG neutron.agent.linux.dhcp [req-7a0b6989-8a94-435e-9762-a778d4869432 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Done building host file /var/lib/neutron/dhcp/4e75ba7d-7b8a-4fed-a0f2-840daba4d743/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-29 02:36:33.122 31040 DEBUG neutron.agent.linux.utils [req-7a0b6989-8a94-435e-9762-a778d4869432 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '2759'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:36:33.136 31040 DEBUG neutron.agent.linux.dhcp [req-7a0b6989-8a94-435e-9762-a778d4869432 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Reloading allocations for network: 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-29 02:36:33.138 31040 DEBUG neutron.agent.linux.utils [req-7a0b6989-8a94-435e-9762-a778d4869432 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743', 'ip', '-4', 'route', 'list', 'dev', 'tap2bfa1d2e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:36:33.152 31040 DEBUG neutron.agent.linux.utils [req-7a0b6989-8a94-435e-9762-a778d4869432 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743', 'ip', '-6', 'route', 'list', 'dev', 'tap2bfa1d2e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:36:33.168 31040 DEBUG neutron.agent.linux.utils [req-7a0b6989-8a94-435e-9762-a778d4869432 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Unable to access /var/lib/neutron/external/pids/4e75ba7d-7b8a-4fed-a0f2-840daba4d743.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-29 02:36:33.169 31040 DEBUG neutron.agent.linux.utils [req-7a0b6989-8a94-435e-9762-a778d4869432 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Unable to access /var/lib/neutron/external/pids/4e75ba7d-7b8a-4fed-a0f2-840daba4d743.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-29 02:36:33.170 31040 DEBUG neutron.agent.linux.external_process [req-7a0b6989-8a94-435e-9762-a778d4869432 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] No process started for 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-29 02:36:33.171 31040 DEBUG oslo_concurrency.lockutils [req-7a0b6989-8a94-435e-9762-a778d4869432 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Releasing semaphore "dhcp-agent-network-lock-4e75ba7d-7b8a-4fed-a0f2-840daba4d743" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-29 02:37:21.543 31040 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-29 02:37:21.554 31040 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.010s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-29 02:37:28.617 31040 DEBUG oslo_concurrency.lockutils [req-1fce6646-2664-47c8-8be7-b0f6bd6cbc59 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Acquired semaphore "dhcp-agent-network-lock-4e75ba7d-7b8a-4fed-a0f2-840daba4d743" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-29 02:37:28.618 31040 DEBUG neutron.agent.dhcp.agent [req-1fce6646-2664-47c8-8be7-b0f6bd6cbc59 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Calling driver for network: 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-29 02:37:28.620 31040 DEBUG neutron.agent.linux.utils [req-1fce6646-2664-47c8-8be7-b0f6bd6cbc59 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743', 'dhcp_release', 'tap2bfa1d2e-58', '11.156.1.1', 'fa:16:3e:17:b9:55'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:37:28.645 31040 DEBUG neutron.agent.linux.dhcp [req-1fce6646-2664-47c8-8be7-b0f6bd6cbc59 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Building host file: /var/lib/neutron/dhcp/4e75ba7d-7b8a-4fed-a0f2-840daba4d743/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-29 02:37:28.650 31040 DEBUG neutron.agent.linux.dhcp [req-1fce6646-2664-47c8-8be7-b0f6bd6cbc59 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Done building host file /var/lib/neutron/dhcp/4e75ba7d-7b8a-4fed-a0f2-840daba4d743/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-29 02:37:28.657 31040 DEBUG neutron.agent.linux.utils [req-1fce6646-2664-47c8-8be7-b0f6bd6cbc59 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '2759'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:37:28.672 31040 DEBUG neutron.agent.linux.dhcp [req-1fce6646-2664-47c8-8be7-b0f6bd6cbc59 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Reloading allocations for network: 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-29 02:37:28.674 31040 DEBUG neutron.agent.linux.utils [req-1fce6646-2664-47c8-8be7-b0f6bd6cbc59 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743', 'ip', '-4', 'route', 'list', 'dev', 'tap2bfa1d2e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:37:28.691 31040 DEBUG neutron.agent.linux.utils [req-1fce6646-2664-47c8-8be7-b0f6bd6cbc59 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743', 'ip', '-6', 'route', 'list', 'dev', 'tap2bfa1d2e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:37:28.707 31040 DEBUG neutron.agent.linux.utils [req-1fce6646-2664-47c8-8be7-b0f6bd6cbc59 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Unable to access /var/lib/neutron/external/pids/4e75ba7d-7b8a-4fed-a0f2-840daba4d743.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-29 02:37:28.708 31040 DEBUG neutron.agent.linux.utils [req-1fce6646-2664-47c8-8be7-b0f6bd6cbc59 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Unable to access /var/lib/neutron/external/pids/4e75ba7d-7b8a-4fed-a0f2-840daba4d743.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-29 02:37:28.709 31040 DEBUG neutron.agent.linux.external_process [req-1fce6646-2664-47c8-8be7-b0f6bd6cbc59 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] No process started for 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-29 02:37:28.710 31040 DEBUG oslo_concurrency.lockutils [req-1fce6646-2664-47c8-8be7-b0f6bd6cbc59 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Releasing semaphore "dhcp-agent-network-lock-4e75ba7d-7b8a-4fed-a0f2-840daba4d743" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-29 02:37:54.160 31040 DEBUG oslo_concurrency.lockutils [req-006ee360-e00b-4da1-9001-87c6c424027d c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Acquired semaphore "dhcp-agent-network-lock-4e75ba7d-7b8a-4fed-a0f2-840daba4d743" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-29 02:37:54.162 31040 DEBUG neutron.agent.dhcp.agent [req-006ee360-e00b-4da1-9001-87c6c424027d c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Calling driver for network: 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 action: disable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-29 02:37:54.164 31040 DEBUG neutron.agent.linux.utils [req-006ee360-e00b-4da1-9001-87c6c424027d c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Running command (rootwrap daemon): ['kill', '-9', '2759'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:37:54.176 31040 DEBUG neutron.agent.linux.utils [req-006ee360-e00b-4da1-9001-87c6c424027d c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-06-29 02:37:54.196 31040 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=br-int, port=tap2bfa1d2e-58) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-29 02:37:54.270 31040 DEBUG neutron.agent.linux.utils [req-029f0c79-1a72-4737-a8ec-b6dd9c40026a - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:37:54.281 31040 DEBUG neutron.agent.linux.utils [req-029f0c79-1a72-4737-a8ec-b6dd9c40026a - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'delete', 'qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:37:54.291 31040 DEBUG oslo_concurrency.lockutils [req-029f0c79-1a72-4737-a8ec-b6dd9c40026a - - - - -] Releasing semaphore "dhcp-agent-network-lock-4e75ba7d-7b8a-4fed-a0f2-840daba4d743" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-29 02:38:00.055 31040 DEBUG neutron.agent.dhcp.agent [req-6f243ef8-598f-4d00-a106-09d807888ae1 - - - - -] resync (4e75ba7d-7b8a-4fed-a0f2-840daba4d743): ['Agent port was deleted'] _periodic_resync_helper /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:255
2018-06-29 02:38:00.057 31040 INFO neutron.agent.dhcp.agent [req-6f243ef8-598f-4d00-a106-09d807888ae1 - - - - -] Synchronizing state
2018-06-29 02:38:01.223 31040 INFO neutron.agent.dhcp.agent [req-37f01664-b66d-4a44-bcd4-d7ea96b56e9a - - - - -] All active networks have been fetched through RPC.
2018-06-29 02:38:01.224 31040 INFO neutron.agent.dhcp.agent [-] Starting network 3a89a4af-574f-492a-9eb4-eae63d51f625 dhcp configuration
2018-06-29 02:38:01.225 31040 INFO neutron.agent.dhcp.agent [-] Finished network 3a89a4af-574f-492a-9eb4-eae63d51f625 dhcp configuration
2018-06-29 02:38:01.226 31040 INFO neutron.agent.dhcp.agent [-] Starting network 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 dhcp configuration
2018-06-29 02:38:01.226 31040 DEBUG neutron.agent.dhcp.agent [-] Calling driver for network: 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 action: enable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-29 02:38:01.228 31040 DEBUG neutron.agent.linux.utils [-] Unable to access /var/lib/neutron/dhcp/4e75ba7d-7b8a-4fed-a0f2-840daba4d743/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-29 02:38:01.228 31040 DEBUG neutron.agent.linux.dhcp [-] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-4e75ba7d-7b8a-4fed-a0f2-840daba4d743 on network 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 does not yet exist. Checking for a reserved port. _setup_reserved_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1267
2018-06-29 02:38:01.229 31040 DEBUG neutron.agent.linux.dhcp [-] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-4e75ba7d-7b8a-4fed-a0f2-840daba4d743 on network 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 does not yet exist. Creating new one. _setup_new_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1288
2018-06-29 02:38:04.639 31040 DEBUG neutron.agent.linux.dhcp [req-f5842c14-1b77-4ab6-9b3a-0ab21927e829 - - - - -] Requested DHCP port with IPs on subnets set([u'f5f0c563-7abb-4002-a413-730683323181']) but only got IPs on subnets set([]). setup_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1341
2018-06-29 02:38:04.640 31040 DEBUG neutron.agent.linux.dhcp [req-f5842c14-1b77-4ab6-9b3a-0ab21927e829 - - - - -] Cleaning stale devices for network 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 _cleanup_stale_devices /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1369
2018-06-29 02:38:04.641 31040 DEBUG neutron.agent.linux.utils [req-f5842c14-1b77-4ab6-9b3a-0ab21927e829 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:38:04.657 31040 DEBUG neutron.agent.dhcp.agent [req-f5842c14-1b77-4ab6-9b3a-0ab21927e829 - - - - -] Error configuring DHCP port, scheduling resync: Subnet on port 7ac65168-dc7b-49f6-b74e-898fba32d773 does not match the requested subnet f5f0c563-7abb-4002-a413-730683323181. call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:153
2018-06-29 02:38:04.658 31040 INFO neutron.agent.dhcp.agent [req-f5842c14-1b77-4ab6-9b3a-0ab21927e829 - - - - -] Finished network 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 dhcp configuration
2018-06-29 02:38:04.659 31040 INFO neutron.agent.dhcp.agent [req-37f01664-b66d-4a44-bcd4-d7ea96b56e9a - - - - -] Synchronizing state complete
2018-06-29 02:38:04.660 31040 DEBUG oslo_concurrency.lockutils [req-f5842c14-1b77-4ab6-9b3a-0ab21927e829 - - - - -] Acquired semaphore "dhcp-agent-network-lock-4e75ba7d-7b8a-4fed-a0f2-840daba4d743" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-29 02:38:04.661 31040 INFO neutron.agent.dhcp.agent [req-f5842c14-1b77-4ab6-9b3a-0ab21927e829 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-29T00:38:01Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-4e75ba7d-7b8a-4fed-a0f2-840daba4d743, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'f5f0c563-7abb-4002-a413-730683323181', u'ip_address': u'11.156.1.2'}], id=7ac65168-dc7b-49f6-b74e-898fba32d773, mac_address=fa:16:3e:46:7e:d7, name=, network_id=4e75ba7d-7b8a-4fed-a0f2-840daba4d743, port_security_enabled=False, project_id=2bae9361f8ba48608796babaf1c6b310, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=2bae9361f8ba48608796babaf1c6b310, updated_at=2018-06-29T00:38:02Z
2018-06-29 02:38:04.662 31040 DEBUG oslo_concurrency.lockutils [req-f5842c14-1b77-4ab6-9b3a-0ab21927e829 - - - - -] Releasing semaphore "dhcp-agent-network-lock-4e75ba7d-7b8a-4fed-a0f2-840daba4d743" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-29 02:38:04.663 31040 DEBUG oslo_concurrency.lockutils [req-b0df3f97-7ff4-4569-b12a-804d0123c3f6 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Acquired semaphore "dhcp-agent-network-lock-4e75ba7d-7b8a-4fed-a0f2-840daba4d743" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-29 02:38:05.080 31040 DEBUG neutron.agent.linux.utils [req-66456060-364f-4d4c-83ac-8d8ecec76e8a - - - - -] Unable to access /var/lib/neutron/external/pids/4e75ba7d-7b8a-4fed-a0f2-840daba4d743.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-29 02:38:05.081 31040 DEBUG neutron.agent.linux.utils [req-66456060-364f-4d4c-83ac-8d8ecec76e8a - - - - -] Unable to access /var/lib/neutron/external/pids/4e75ba7d-7b8a-4fed-a0f2-840daba4d743.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-29 02:38:05.082 31040 DEBUG neutron.agent.linux.external_process [req-66456060-364f-4d4c-83ac-8d8ecec76e8a - - - - -] No process started for 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-29 02:38:05.083 31040 DEBUG neutron.agent.dhcp.agent [req-66456060-364f-4d4c-83ac-8d8ecec76e8a - - - - -] Calling driver for network: 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 action: disable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-29 02:38:05.093 31040 DEBUG neutron.agent.linux.utils [req-66456060-364f-4d4c-83ac-8d8ecec76e8a - - - - -] Unable to access /var/lib/neutron/dhcp/4e75ba7d-7b8a-4fed-a0f2-840daba4d743/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-29 02:38:05.096 31040 DEBUG neutron.agent.linux.utils [req-66456060-364f-4d4c-83ac-8d8ecec76e8a - - - - -] Unable to access /var/lib/neutron/dhcp/4e75ba7d-7b8a-4fed-a0f2-840daba4d743/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-29 02:38:05.096 31040 DEBUG neutron.agent.linux.external_process [req-66456060-364f-4d4c-83ac-8d8ecec76e8a - - - - -] No process started for 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-29 02:38:05.098 31040 DEBUG neutron.agent.linux.dhcp [req-66456060-364f-4d4c-83ac-8d8ecec76e8a - - - - -] Impossibile accedere a /var/lib/neutron/dhcp/4e75ba7d-7b8a-4fed-a0f2-840daba4d743/interface _get_value_from_conf_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:268
2018-06-29 02:38:05.099 31040 DEBUG neutron.agent.linux.dhcp [req-66456060-364f-4d4c-83ac-8d8ecec76e8a - - - - -] No interface exists for network 4e75ba7d-7b8a-4fed-a0f2-840daba4d743 destroy /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1482
2018-06-29 02:38:06.741 31040 DEBUG neutron.agent.linux.utils [req-0f9d49cb-07d2-4da7-85fb-f52255ae075e - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-29 02:38:06.759 31040 DEBUG neutron.agent.linux.dhcp [req-0f9d49cb-07d2-4da7-85fb-f52255ae075e - - - - -] Namespace already deleted: qdhcp-4e75ba7d-7b8a-4fed-a0f2-840daba4d743 _destroy_namespace_and_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:246
2018-06-29 02:38:06.761 31040 DEBUG oslo_concurrency.lockutils [req-0f9d49cb-07d2-4da7-85fb-f52255ae075e - - - - -] Releasing semaphore "dhcp-agent-network-lock-4e75ba7d-7b8a-4fed-a0f2-840daba4d743" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-29 02:38:06.762 31040 DEBUG oslo_concurrency.lockutils [req-f5842c14-1b77-4ab6-9b3a-0ab21927e829 - - - - -] Acquired semaphore "dhcp-agent-network-lock-4e75ba7d-7b8a-4fed-a0f2-840daba4d743" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-29 02:38:06.763 31040 DEBUG oslo_concurrency.lockutils [req-f5842c14-1b77-4ab6-9b3a-0ab21927e829 - - - - -] Releasing semaphore "dhcp-agent-network-lock-4e75ba7d-7b8a-4fed-a0f2-840daba4d743" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-29 02:38:08.236 31040 DEBUG oslo_concurrency.lockutils [req-b0df3f97-7ff4-4569-b12a-804d0123c3f6 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Acquired semaphore "dhcp-agent-network-lock-4e75ba7d-7b8a-4fed-a0f2-840daba4d743" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-29 02:38:08.237 31040 DEBUG oslo_concurrency.lockutils [req-b0df3f97-7ff4-4569-b12a-804d0123c3f6 c5cd6ef06ce14dd690705b39bb3f1552 2776224a461d4a01833ab75847b98330 - - -] Releasing semaphore "dhcp-agent-network-lock-4e75ba7d-7b8a-4fed-a0f2-840daba4d743" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-29 02:38:21.543 31040 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-29 02:38:21.548 31040 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.005s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-29 02:38:34.660 31040 DEBUG neutron.agent.dhcp.agent [req-37f01664-b66d-4a44-bcd4-d7ea96b56e9a - - - - -] resync (4e75ba7d-7b8a-4fed-a0f2-840daba4d743): [SubnetMismatchForPort(u'Subnet on port 7ac65168-dc7b-49f6-b74e-898fba32d773 does not match the requested subnet f5f0c563-7abb-4002-a413-730683323181.',), 'Agent port was modified'] _periodic_resync_helper /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:255
2018-06-29 02:38:34.662 31040 INFO neutron.agent.dhcp.agent [req-37f01664-b66d-4a44-bcd4-d7ea96b56e9a - - - - -] Synchronizing state
2018-06-29 02:38:35.617 31040 INFO neutron.agent.dhcp.agent [req-56806c35-3696-4b67-b36a-80c6450dd0aa - - - - -] All active networks have been fetched through RPC.
2018-06-29 02:38:35.619 31040 INFO neutron.agent.dhcp.agent [-] Starting network 3a89a4af-574f-492a-9eb4-eae63d51f625 dhcp configuration
2018-06-29 02:38:35.621 31040 INFO neutron.agent.dhcp.agent [-] Finished network 3a89a4af-574f-492a-9eb4-eae63d51f625 dhcp configuration
2018-06-29 02:38:35.622 31040 INFO neutron.agent.dhcp.agent [req-56806c35-3696-4b67-b36a-80c6450dd0aa - - - - -] Synchronizing state complete
